backend/backend/JwtAuthentication/JwtAuthentication.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+
+namespace backend.JwtAuthentication
+{
+    public class JwtAuthentication : IJwtAuthentication
+    {
+        private readonly BackendContext _db;
+        private readonly string _tokenKey;
+
+        public JwtAuthentication(string tokenKey, BackendContext context)
+        {
+            _tokenKey = tokenKey;
+            _db = context;
+        }
+
+        public string Authenticate(string email, string password)
+        {
+            if (!_db.Users.Any(u => u.Password == password && u.Email == email))
```

Comment:
The password should be hashed. I don't know if we're gonna do it later, or now but `password` should be hashed before comparing. Hashing will probably be implemented alongside 'Register' though, so maybe we'll just add it later here.

Reply:
We have another task for password encryption. This is just a "sample" implementation of authentication